complex.c

```diff
@@ -1365,7 +1365,7 @@ rb_complex_infinite_p(VALUE self)
     if (RB_FLOAT_TYPE_P(magnitude)) {
 	const double f = RFLOAT_VALUE(magnitude);
 	if (isinf(f)) {
-	    return INT2FIX(f < 0 ? -1 : 1);
+	    return INT2FIX(1);
```

Comment:
You can use ``ONE`` defined at the top instead of ``INT2FIX(1)``.

Reply:
Thanks! I fixed.